docs/site/content/docs/latest/ref-unmanaged-cluster.md

```diff
@@ -0,0 +1,201 @@
+# Unmanaged Clusters Reference
```

Comment:
TODO(@joshrosso): Add content in for switching contexts between clusters.

Reply:
going to keep this out of the docs until we implement a context-switch capability.